packages/suite/src/components/guide/GuidePanel.tsx

```diff
@@ -42,44 +75,59 @@ const GuidePanel = () => {
 
     // if guide is open, do not animate guide opening if transitioning between onboarding, welcome and suite layout
     const [guideAlreadyOpen, setGuideAlreadyOpen] = useState(guideOpen);
+
+    const { close } = useActions({
```

Comment:
You can take `closeGuide` from `useGuide` above

Reply:
done